planetiler-custommap/src/main/java/com/onthegomap/planetiler/custommap/configschema/ZoomOverride.java

```diff
@@ -0,0 +1,12 @@
+package com.onthegomap.planetiler.custommap.configschema;
+
+import java.util.Map;
+
+/**
+ * Configuration item that instructs the renderer to override the default zoom range for features which contain specific
+ * tag combinations.
+ */
+public record ZoomOverride(
+  Byte min,
+  Byte max,
```

Comment:
bytes are a little inconvenent to work with, and we're not short on space, could we just use `Integer` ?

Reply:
Fixed in be4fafb